apps/foundation-demo/src/app/checkout/components/shipping/shipping-options/shipping-options.component.html

```diff
@@ -1,7 +1,7 @@
 <fieldset class="shipping-options">
   <legend class="shipping-options__title">Shipping Options</legend>
-  <div class="shipping-options__radio" *ngFor="let option of shippingOptions" (click)="onSelectShippingOption(option.id)">
```

Comment:
Why are we emitting `text` instead of `id`

Reply:
You commented this on the https://github.com/graycoreio/daffodil/pull/45 review, so I'll address it there

Reply:
Ok. Then ignore this.